docs/MainSite/ReportingGuide.md

```diff
@@ -0,0 +1,91 @@
+include as much information as you can. If you feel comfortable,
+please consider including:
+- Your contact info (so we can get in touch with you if we need to
+  follow up).
+- Names (real, nicknames, or pseudonyms) of any individuals
+involved. If there were other witnesses besides you, please try to
+include them as well.
+- When and where the incident occurred. Please be as specific as
+  possible.
+- Your account of what occurred. If there is a publicly available
+record (e.g. a mailing list archive or Slack logs) please include a
+link.
+- Any extra context you believe existed for the incident.
+- If you believe this incident is ongoing.
+- Any other information you believe we should have.
+
+### What happens after you file a report?
+
+You will receive an email from the Code of Conduct Committee or
+project leader(s) you contacted, acknowledging receipt within 24 hours
+(and we will aim to respond much quicker than that). They will review
+the incident and try to determine:
+- What happened and who was involved.
+- Whether this event constitutes a code of conduct violation.
+- Whether this is an ongoing situation, or if there is a threat to
+  anyone’s physical safety.
+
+Once the Code of Conduct Committee or contacted project leader(s) have
+a complete account of the events they will make a decision on how to
+respond, possibly in consultation with the full Executive Committee
+(excluding anyone involved in the incident). Responses may include:
+- Nothing, if we determine no violation occurred or it has already
+  been appropriately resolved.
+- Providing either moderation or mediation to ongoing interactions
+  (where appropriate, safe, and desired by both parties).
+- A private reprimand from the working group to the individuals
+  involved.
+- An imposed vacation (i.e. asking someone to “take a week off” from a
+  mailing list or Slack).
+- Escalation to the appropriate institutions.
+- Involvement of relevant law enforcement if appropriate.
+
+If the situation is not resolved within one week, we’ll respond within
+one week to the original reporter with an update and explanation.
+Once we’ve determined our response, we will separately contact the
+original reporter and other individuals to let them know what actions
+(if any) we’ll be taking. We will take into account feedback from the
+individuals involved on the appropriateness of our response, but we
+don’t guarantee we’ll act on it.
+
+### Code of Conduct Committee
+
+The current members of the Code of Conduct Committee (and the only
+people that see emails sent to conduct@spectre-code.org) are:
+- TBD: Saul Teukolsky
+- TBD: Harald Pfieffer
+- TBD: ???
+
+SpECTRE is being developed in support of our collaborative Simulating
+eXtreme Spacetimes (SXS) research program into the multi-messenger
+astrophysics of neutron star mergers, core-collapse supernovae, and
+gamma-ray bursts.  As such, it falls under the oversight of the SXS
+Executive Committee whose current members are:
```

Comment:
I think this statement is trying to say that SXS oversees the code, and I think that's fair, since for now everyone involved is an SXS member. Since SXS is governed by the exec committee, that's the group that takes oversight overall. I don't think this is intended to imply technical oversight, which is from the core developers, but instead it is meant to imply overall management. This is a good point to discuss, though!

Reply:
I understood that, but there are people on the list that again have as far as I have been able to tell had no managerial involvement in SpECTRE. I'm just not entirely sure what this paragraph and list are trying to achieve, which is probably what is confusing me.